pinot-core/src/test/java/org/apache/pinot/queries/InterSegmentAggregationMultiValueQueriesTest.java

```diff
@@ -57,6 +60,82 @@ public void testCountMV() {
     brokerResponse = getBrokerResponseForPqlQuery(query + MV_GROUP_BY);
     QueriesTestUtils
         .testInterSegmentAggregationResult(brokerResponse, 400000L, 0L, 800000L, 400000L, new String[]{"199896"});
+
+    query = "SELECT COUNTMV(column6) FROM testTable GROUP BY VALUEIN(column7, 363, 469, 246, 100000)";
+    brokerResponse = getBrokerResponseForPqlQuery(query);
+    List<String[]> groupKeys = new ArrayList<>();
+    groupKeys.add(new String[]{"363"});
```

Comment:
It will be useful to have these numbers defined as key1, key2, value1, value2 ,etc. WIll make it easier to extend the test if we want to.